Mathlib/CategoryTheory/Sites/Coherent.lean

```diff
@@ -82,4 +83,148 @@ lemma isSheaf_coherent [Precoherent C] (P : Cᵒᵖ ⥤ Type w) :
     rintro B S ⟨α, _, X, π, rfl, hS⟩
     exact h _ _ _ _ hS
 
+
+namespace coherentTopology
+
+variable {C : Type _} [Category C] [Precoherent C]
+
+variable {X : C}
+/--
+For a precoherent category, any sieve that contains an `EffectiveEpiFamily` is a sieve of the
+coherent topology.
+Note: This is one direction of `Sieve_iff_hasEffectiveEpiFamily`, but is needed for the proof.
+-/
+theorem Sieve_of_hasEffectiveEpiFamily (S : Sieve X) :
```

Comment:
Should it be `GrothendieckTopology.mem_sieves_of_...`?

Reply:
That makes sense, thanks. Renamed this and the other theorem to that extent.